positron/comms/data_explorer-backend-openrpc.json

```diff
@@ -47,7 +47,7 @@
 				{
 					"name": "start_index",
 					"description": "Index (starting from zero) of first result to fetch (for paging)",
-					"required": false,
+					"required": true,
```

Comment:
Yuup. These two changes are because `generate-comms.ts` wasn't previously handling this field so all of our code assumes that these are actually required.

Reply:
I also wish JSON could have comments. :-(